src/main/java/com/google/flourbot/datastorage/FirebaseDataStorage.java

```diff
@@ -46,7 +46,7 @@ public FirebaseDataStorage() {
     Query query =
         db.collection(COLLECTION_NAME)
             .whereEqualTo(MACRO_IDENTIFIER, macroName)
-            .whereArrayContains(SCOPE_IDENTIFIER, userEmail);
+            .whereEqualTo(CREATOR_IDENTIFIER, userEmail);
```

Comment:
Are we not using the list scope anymore?

Reply:
I am not sure, personally I don't know how to make them work together. Also, if we have the room sharing feature working, I am not sure what capabilities the scope field adds. What do you think?

Reply:
Yep just making sure! I like the initialize approach better and I envision that working well with the `Team` structure as well.

Reply:
Ok great! I'll ping you when I've merged this to master. Will you be able to remove the scope input in the website @tsa87?